code/game/machinery/suit_cycler.dm

```diff
@@ -458,21 +458,15 @@
 /obj/machinery/suit_cycler/proc/apply_paintjob()
 	if(!target_bodytype || !target_modification)
 		return
-	var/target_flags = available_bodytypes[target_bodytype]
-	if(helmet) helmet.refit_for_bodytype(target_flags)
-	if(suit)   suit.refit_for_bodytype(target_flags)
-	if(boots)  boots.refit_for_bodytype(target_flags)
-
 
 	if(helmet)
 		target_modification.RefitItem(helmet)
 		helmet.refit_for_bodytype(target_bodytype)
+		helmet.SetName("refitted [helmet.name]")
```

Comment:
If I refit it repeatedly won't this give me `refitted refitted blah` and so on? Maybe `initial`, or there's some fancy extension thing I believe.

Reply:
`RefitItem` already sets the item name to a base name.

Reply:
In that case, should probably change boots below, which, strangely, are not copy-paste but use `initial`.

Reply:
Magboots are not visually refitted currently as there are no voidsuit-specific booties.